packages/firestore/GEMINI.md

```diff
@@ -0,0 +1,5 @@
+# Firestore JavaScript SDK
+This folder contains the authoritative and official JavaScript SDK for the [Google Cloud Firestore](https://firebase.google.com/docs/firestore) database.
```

Comment:
JavaScript Web SDK - "Web" to differentiate from Admin and Cloud JS SDKs.

It supports building client apps for the Browser, Node, React Native, and more.